src/main/java/org/openlmis/requisition/domain/RequisitionStatus.java

```diff
@@ -19,6 +19,7 @@
 
 public enum RequisitionStatus {
   INITIATED(1),
+  REJECTED(1),
```

Comment:
I'm ok with that, wonder if we should mark in approval and submitted with the same val?